templates/types/streaming/fastapi/app/engine/utils/file_helper.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import os
+import uuid
+from typing import Optional
+
+from pydantic import BaseModel
+
+logger = logging.getLogger(__name__)
+
+
+class FileMetadata(BaseModel):
+    outputPath: str
+    filename: str
+    url: str
+
+
+def save_file(
```

Comment:
i guess the goal is to replace every call that is using FILESERVER_URL_PREFIX with the method?

Reply:
Yes, that was the idea: to save a file and get a URL for the file. But I thought it would be better for another PR because it will need to refactor some other code a bit to use this.

Reply:
Great. Please keep that in mind.